app/assets/javascripts/components/activity/activity_table_row.jsx

```diff
@@ -65,7 +65,9 @@ const ActivityTableRow = React.createClass({
 
     let diffViewer;
     if (this.props.revision && this.props.revision.api_url) {
-      diffViewer = <DiffViewer revision={this.props.revision} />;
+      diffViewer = <DiffViewer revision={this.props.revision}
```

Comment:
the `revision` object already includes the character count, so there's no need to pass it as a separate prop. Within the DiffViewer, you can do `this.props.revision.characters`.